v2.0/explain.md

```diff
@@ -4,10 +4,32 @@ summary: The EXPLAIN statement provides information you can use to optimize SQL
 toc: false
 ---
 
-The `EXPLAIN` [statement](sql-statements.html) returns CockroachDB's query plan to execute [`DELETE`](delete.html), [`INSERT`](insert.html), [`SELECT`](select.html) or [`UPDATE`](update.html) statements. You can then use this information to optimize those queries.
+The `EXPLAIN` [statement](sql-statements.html) returns CockroachDB's query plan to execute the [explainable statements](#explainable-statments). You can then use this information to optimize those queries.
 
 <div id="toc"></div>
 
+## Explainable Statements
+
```

Comment:
Let's us an intro sentence like you have in the 1.1 version of the page:

> You can use `EXPLAIN` on the following statements:

Reply:
done